### PR TITLE
translation(chirag): add bitflip english doc

### PR DESCRIPTION
**Pull request by deepNet-Chirag**
## Pull request for translation of documents in English

- [x] bitfilp

### CODE REVIEW

**Comment by PaParaZz1** [comment]
Please modify the name of pull request as our format

File: source/13_envs/bitflip.rst [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+Installation Method
+--------
+
+The BitFlip environment does not need to be installed, it is built into DI-engine.
+
+Runnable Code Example in DI-zoo
+========================
+
+Below is a complete Bitflip environment config that uses the DQN algorithm as the policy. Please run the "bitflip_dqn_main.py" file in the "\DI-engine\dizoo\classic_control\bitflip\entry" directory as follows.
```
[the remark this answers is not in the document]
Comment (PaParaZz1) [reply source/13_envs/bitflip.rst:23]: @deepNet-Chirag Maybe you should modify the description like what @zjowowen said

File: source/13_envs/bitflip.rst [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+Overview
+=======
+BitFlip is a very simple little game. Assuming there are n coins, each coin has two states, the positive side is denoted as 0 and the negative side is denoted as 1. The action space is a vector of length n, and executing the nth action type represents flipping the nth coin.
+For each episode, we randomly initialize the coin state and target state. If the coin state and the target state are not the same, the reward is -1, otherwise it is 1.
+
+.. image:: ./images/bitflip.gif
+   :align: center
+
+Installation
+====
```
Comment (PaParaZz1) [inline source/13_envs/bitflip.rst:13]: the notation should be longer than above text

**Review by PaParaZz1 — APPROVED** [review]
This PR can be merged after solving the comments

**Comment by PaParaZz1** [comment]
@deepNet-Chirag Do you have time to polish this PR according to review comments?

**Comment by deepNet-Chirag** [comment]
Yeah! My apologies as I was out station. Working on it Asap!

File: source/13_envs/bitflip.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (deepNet-Chirag) [reply source/13_envs/bitflip.rst:23]: Yeah

**Comment by deepNet-Chirag** [comment]
I think it's done now @PaParaZz1

**Pull request merged**